### PR TITLE
Upgrade bs-platform to 4.x and reason-react to 0.5.x

### LINKED ISSUE #3: Upgrade to bs-platform 3.1.0 to avoid warnings
Currently there is warning about JS.boolean, JS.false_, Js.Undefined.from_opt. Would be nice to adjust that. I can make a PR if you are interested.

### PR DESCRIPTION
**Pull request by MoOx**
Easy upgrade, mostly to avoid warnings.

Closes #3

### CODE REVIEW

File: package.json
Comment (MoOx) [inline package.json]: BTW, maybe all of this deps should be peerDependencies? Thoughts?

**Comment by MoOx** [comment]
I updated the PR to bs-platform 4.x and reason-react 0.5.x

**Comment by MoOx** [comment]
@bguzryanto Are you open to you add me as a maintainer so I can handle this easy upgrades?

**Comment by bguzryanto** [comment]
@MoOx yes open for maintainer! Added you :)

**Comment by MoOx** [comment]
Could you also add me as `moox` on npm so I can cut a release later (after merging pending work)? Thanks!

**Review by bguzryanto — COMMENTED** [review]
LGTM

**Review by bguzryanto — APPROVED** [review]

**Comment by bguzryanto** [comment]
You can merge it first I will add you too to NPM later this night, SGT :)

**Pull request merged**
